components/global/global_header.scss

```diff
@@ -7,6 +7,12 @@
     --global-header-background-rgb: var(--sidebar-teambar-bg-rgb);
     --global-header-text-rgb: var(--sidebar-header-text-color-rgb);
 
+    @media screen and (max-width: 768px) {
```

Comment:
hiding the global header VS not even rendering it is a discussion that could be held here. IMO the "not rendering" (aka `Utils.isMobile() === true`) approach is the better solution, since renderin it in the first place will not only render the global header itself, but all of its subcomponents as well. Correct me if I am wrong, but this can have an impact on performance.

Reply:
as to your reasoning in the PR description why this should render nevertheless: It is correct, but the check could also be bound to the resizeEvent, so that it is performed (maybe in conjunction with a debounce to not have it hit performance). That way we can have the best from both approaches! :D

Reply:
I agree it's not ultimately the best approach. The challenge is that as is, the `const isMobileView = UserAgent.isMobile() || Utils.isMobile();` check causes the following bugs:

1. iOS/Android Tablets with a screen width `>768px` won't get the GH experience when they should be
2. Resizing the screen from `>768px` to `<=768px` or the other way doesn't properly re-render the UI creating user experience issues that in some cases can only be recovered from by reloading.

**1** could be partially corrected by just using `Utils.isMoblie()` and not `UserAgent.isMobile()`, however window resizing would still likely have issues in the current implementation.
**2** and the resizing issue of **1** could be fixed by implementing redux-enabled functionality (or similar) that properly monitors screen resizing and updates a new `isMobile()` setting that could replace/augment the existing isMobile checks

Considerations:
1. Not showing GH on screen sizes `<=768px` is temporary
2. Implementing a redux solution (or similar) will require additional effort to build, review and test and generally make sure the overall experience is as expected.
3. The markup for the initial state of the GH is minimal and with `display: none` isn't processed by the browser during rendering
4. We have very limited time left
5. CSS is a much simpler approach for the short term

If you'd like to tackle an updated responsive JS solution on top of your [previous PR](https://github.com/mattermost/mattermost-webapp/pull/8561) that handles the above bugs in the time we have left (keeping in mind PR reviews and QA), then I'm happy to remove the CSS solution in this ticket. Let me know. :)